plugins/woocommerce/tests/api-core-tests/README.md

```diff
@@ -127,25 +129,279 @@ After you run a test, it's best to restart the environment to start from a fresh
 
 ## Guide for writing tests
 
+When writing new tests, a good source on how to start writing and how to wire things up is to reference the other existing tests (found in the [tests](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/api-core-tests/tests) folder). Any data required for these tests should be located in an equivalent file in the [data](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/api-core-tests/data) folder.
```

Comment:
Maybe here we can consider including some "good examples" (1-2) so folks have a good starting point when jumping in. We could include a test and how it uses the equivalent data file.

Minor wording change suggestions:

```suggestion
When writing new tests, a good source on how to get started is to reference the [existing tests](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/api-core-tests/tests). Data that is required for the tests should be located in an equivalent file in the [data](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/api-core-tests/data) folder.
```

Reply:
> Maybe here we can consider including some "good examples" (1-2) so folks have a good starting point when jumping in. We could include a test and how it uses the equivalent data file.
> 
Great suggestion - I'll get this in